dotnet/test/Azure.Iot.Operations.Protocol.IntegrationTests/MemMonEnvoyTests.cs

```diff
@@ -129,7 +129,7 @@ public async Task Send_ReceiveTelemetryWithMetadataAndCE()
         var ManagedMemoryCorrelationId = Guid.NewGuid();
         var ManagedMemoryUserDataKey = Guid.NewGuid().ToString();
         var ManagedMemoryUserDataValue = Guid.NewGuid().ToString();
-        var ManagedMemoryTelemetryMetadata = new OutgoingTelemetryMetadata() { CloudEvent = new CloudEvent(new Uri("test://mq")) };
+        var ManagedMemoryTelemetryMetadata = new OutgoingTelemetryMetadata() { CloudEvent = new CloudEvent(new Uri("test://mq"),  "ms.aio.telemetry") };
```

Comment:
would the customer have to know this value to set it, or is this defaulted within telemetry somewhere? (might just be rusty in dotnet)